files/jquery.grid-a-licious/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "Grid-A-Licious",
+  "repo": "suprb/Grid-A-Licious",
+  "files": {
+    "include": ["jquery.grid-a-licious.min.js"]
```

Comment:
No tags here https://github.com/suprb/Grid-A-Licious/releases it will fail to update